scripts/ingests/ingest_jwst_spectra.py

```diff
@@ -0,0 +1,45 @@
+from astrodb_scripts import load_astrodb, ingest_instrument, ingest_publication
```

Comment:
I wonder at the filename `ingest_jwst_spectra` and what it's doing. Is this a script we want to continuously update as we ingest more? 
If this is a one-off, should it be `ingest_jwst_Fahe24` or something like that? 
If it's meant for single spectra and users just change the individual settings, should it be `ingest_single_spectrum`?

Reply:
I will rename the script to be a one off.